Automattic/jetpack
# Jetpack Assistant: Correct purchased product feature class

### Changes proposed in this Pull Request:

* This PR aims to correct a broken class name that was introduce during the main commit of the Product Suggestions step in the Jetpack Assistant (see attached screenshots).

In the current version of the plugin the list is still functional but it does not include a CSS fix to correct a slight offset in purchased product features.

---

### Jetpack product discussion



There is no related conversations about this. I found it while working on related changes.

---

### Does this pull request change what data or activity we track or use?



No.

---

### Testing instructions:






* Start up a Jetpack site and go to `/wp-admin/admin.php?page=jetpack#/recommendations`
* Proceed to go through the recommendations and purchase one of the suggested products on the `#recommendations/product-suggestions` step.
* After purchasing the product you will be redirected to `/wp-admin/admin.php?page=jetpack#/recommendations/product-purchased` where you can find the purchased product list (see attached screenshots).
* The `<li>` feature items should now have `jp-recommendations-product-purchased__feature` instead of `jp-recommenconst ProductPurchasedComponent = props => {dations-product-purchased__feature` as their class.

---

### Screenshots

#### Before

![Screenshot 2021-09-09 at 13 44 28](https://user-images.githubusercontent.com/3846700/132681740-18734934-f62c-42c2-8122-a4c2a2fb9b23.png)

#### After

![Screenshot 2021-09-09 at 13 44 41](https://user-images.githubusercontent.com/3846700/132681750-10601c40-f89b-4c82-a481-91a4ea43073d.png)
